lwt/jbuild

```diff
@@ -0,0 +1,8 @@
+(library
+ ((name        conduit_lwt)
+  (public_name conduit-lwt)
+  (preprocess (pps (ppx_sexp_conv)))
+  (wrapped false)
+  (modules (resolver_lwt))
+  (libraries (conduit lwt))))
+(jbuild_version 1)
```

Comment:
done in 9207e619d33a15be87b04a4a739d660a9be8c789.

@diml does `jbuild_version` really have to be the top line? it would be nice if jbuilder would warn if it appeared in an incorrect place if that's so